benchmarks/benchmarks/spatial.py

```diff
@@ -347,24 +347,26 @@ class Xdist(Benchmark):
     def setup(self, num_points, metric):
         np.random.seed(123)
         self.points = np.random.random_sample((num_points, 3))
-        # use an equal weight vector to satisfy those metrics
-        # that require weights
-        if metric == 'wminkowski':
-            self.w = np.ones(3)
+        if metric == 'minkowski':
+            # Default p=2 is just the euclidean metric
+            self.kwargs = {'p': 3.0}
```

Comment:
Changing this default will make it hard to compare historical performance. Maybe we can add a special case 'minkowski-L3' and have that one add the `p=3` kwarg?

Reply:
I though asv always used the current benchmark source when doing bench-compare?

Even if that's not the case, prior to this change, `p=2` and `p=3` are indistinguishable in performance. So, this shouldn't actually change the result of the benchmark.